admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -9,125 +9,141 @@
+      *}
+    </nav>
+    {if isset($displayBackOfficeTop)}{$displayBackOfficeTop}{/if}
+  </header>
 
-    {* TODO: REPLACE THE AJAX RUNNING SPINNER WITH THE ONE FROM THE UI KIT
-    <span id="ajax_running">
-      <i class="icon-refresh icon-spin icon-fw"></i>
-    </span>
-    *}
-  </nav>
-  {if isset($displayBackOfficeTop)}{$displayBackOfficeTop}{/if}
-</header>
 
-{include file='components/layout/nav_bar.tpl'}
+  {include file='components/layout/nav_bar.tpl'}
 
-<div id="main-div">
+  <div id="main-div">
 
-  {if $install_dir_exists}
+    {if $install_dir_exists}
 
-    <div class="alert alert-warning">
-      {l s='For security reasons, you must also delete the /install folder.'}
-    </div>
+      <div class="alert alert-warning">
+        {l s='For security reasons, you must also delete the /install folder.'}
+      </div>
 
-  {else}
+    {else}
 
-    {if isset($page_header_toolbar)}{$page_header_toolbar}{/if}
-    {if isset($modal_module_list)}{$modal_module_list}{/if}
+      {if isset($page_header_toolbar)}{$page_header_toolbar}{/if}
+      {if isset($modal_module_list)}{$modal_module_list}{/if}
 
-    <div class="content-div {if !isset($page_header_toolbar)}-notoolbar{/if} {if $current_tab_level == 3}with-tabs{/if}">
+      <div class="content-div {if !isset($page_header_toolbar)}-notoolbar{/if} {if $current_tab_level == 3}with-tabs{/if}">
 
-      {hook h='displayAdminAfterHeader'}
+        {hook h='displayAdminAfterHeader'}
 
-      {* TODO: REPLACE THIS ELEMENT
-      {if isset($conf)}
-        <div class="bootstrap">
-          <div class="alert alert-success">
-            <button type="button" class="close" data-dismiss="alert">&times;</button>
-            {$conf}
+        {* TODO: REPLACE THIS ELEMENT
+        {if isset($conf)}
+          <div class="bootstrap">
+            <div class="alert alert-success">
+              <button type="button" class="close" data-dismiss="alert">&times;</button>
+              {$conf}
+            </div>
+          </div>
+        {/if}
+        *}
+
+        {include file='components/layout/error_messages.tpl'}
+        {include file='components/layout/information_messages.tpl'}
+        {include file='components/layout/confirmation_messages.tpl'}
+        {include file='components/layout/warning_messages.tpl'}
+
+        <div class="row ">
+          <div class="col-sm-12">
+            {$page}
+            {hook h='displayAdminEndContent'}
           </div>
         </div>
-      {/if}
-      *}
 
-      {include file='components/layout/error_messages.tpl'}
-      {include file='components/layout/information_messages.tpl'}
-      {include file='components/layout/confirmation_messages.tpl'}
-      {include file='components/layout/warning_messages.tpl'}
-
-      <div class="row ">
-        <div class="col-sm-12">
-          {$page}
-          {hook h='displayAdminEndContent'}
-        </div>
       </div>
 
+    {/if}
+{else}
+  <div id="main-div">
+    <div class="-notoolbar">
```

Comment:
This one is not closed

Reply:
It's on purpose, it's the counterpart of https://github.com/PrestaShop/PrestaShop/blob/28c90fec3869ffb6a98d48b5193bd171d803df50/admin-dev/themes/new-theme/template/layout.tpl#L91 that was being injected in the `{if}` above.

Reply:
:nauseated_face: 
You must review the indentation or better try to fixed it :/